### ISSUE TITLE
affects_stability_of can't make plan

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#68]
```
{'message': {'query_graph': {'nodes': {'a': {'category': 'biolink:NamedThing',
     'id': 'NCBIGENE:18671'},
    'b': {'category': 'biolink:ChemicalSubstance'}},
   'edges': {'ab': {'subject': 'a',
     'object': 'b',
     'predicate': 'biolink:affects_stability_of'}}}}}
```

There's data in automat/ctd.

"Could not find a plan to traverse query graph"

This may be fixed with a rebuild of the registry, it's possible that this is a new predicate ctd...

**Comment by cbizon** [comment]
Same is true of ctd predicate biolink:decreases_mutation_rate_of

```
{'message': {'query_graph': {'nodes': {'a': {'category': 'biolink:ChemicalSubstance',
     'id': 'CHEBI:8069'},
    'b': {'category': 'biolink:Gene'}},
   'edges': {'ab': {'subject': 'a',
     'object': 'b',
     'predicate': 'biolink:decreases_mutation_rate_of'}}}}}
```

**Comment by patrickkwang** [comment]
The first query above appears to now make a call to automat/ctd, but no data is returned. Sending this to automat/ctd directly returns no results:
```
{
    "message": {
        "query_graph": {
            "nodes": {
                "a": {
                    "category": "biolink:Gene",
                    "id": "NCBIGENE:18671"
                },
                "b": {
                    "category": "biolink:ChemicalSubstance"
                }
            },
            "edges": {
                "ab": {
                    "subject": "a",
                    "object": "b",
                    "predicate": "biolink:affects_stability_of"
                }
            }
        }
    }
}
```

**Comment by patrickkwang** [comment]
The second appears fixed as of #83.

**Comment by cbizon** [comment]
OK, sounds like this one can be closed then - I'll check on the CTD side
